library/ssl_tls.c

```diff
@@ -665,6 +665,21 @@ typedef int ssl_tls_prf_t(const unsigned char *, size_t, const char *,
                           const unsigned char *, size_t,
                           unsigned char *, size_t);
 
+static int psa_status_to_mbedtls( psa_status_t status )
```

Comment:
I needed this extra function to keep returned status consistent (psa vs mbedtls). 
The same function is added in `library/ssl_tls13_keys.c`. Is there some common approach for such conversion problem?
There is `mbedtls_to_psa_error()` function that coverts mbedtls to psa status, but it seems that opposite conversion is not implemented.

Reply:
I think the conversing can be quite context-dependent, for example on low-memory we'll want to return `MBEDTLS_ERR_XXX_ALLOC_FAILED` with XXX corresponding to the current module (this is a known issue with the design of our error codes), which is probably why there's not universal function, and each module tends to do its own thing.